visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -2038,7 +2038,7 @@ namespace pcl
           static FPSCallback *New () { return (new FPSCallback); }
 
           FPSCallback () : actor (), pcl_visualizer (), decimated (), last_fps(0.0f) {}
-          FPSCallback (const FPSCallback& src) : actor (src.actor), pcl_visualizer (src.pcl_visualizer), decimated (src.decimated), last_fps (src.last_fps) {}
+          FPSCallback (const FPSCallback& src) : vtkCommand (), actor (src.actor), pcl_visualizer (src.pcl_visualizer), decimated (src.decimated), last_fps (src.last_fps) {}
```

Comment:
Wrong fix: `vtkCommand (src)` is correct.

Reply:
Hm, I just copied whatever was there before #3070:

https://github.com/SunBlack/pcl/blame/8b8ec7b3a762870c73044c47b0cb4a144964ebb9/visualization/include/pcl/visualization/pcl_visualizer.h#L2041

However you are right, should rather be `vtkCommand(src)`.